app/routes/events/components/EventDetail/index.tsx

```diff
@@ -426,11 +426,20 @@ export default class EventDetail extends Component<Props, State> {
               iconName="time-outline"
               content={<FromToTime from={event.startTime} to={event.endTime} />}
             />
+            {event.isForeignLanguage !== null && (
```

Comment:
I think it'd look nicer to not display this text field at all if `isForeignLanguage` is false. When 95% of events are in norwegian, it'd look far less cluttered.